designer/client/src/http/HttpService.ts

```diff
@@ -423,11 +423,15 @@ class HttpService {
         expression: Expression,
         caretPosition2d: CaretPosition2d,
         variables: Record<string, any>,
+        scenarioName: string,
+        processProperties: PropertiesType,
```

Comment:
With six parameters I think we hit the critical point - maybe it's time to create `ExpressionSuggestionRequest`?

Reply:
Good idea, changed